tests/test_logical_types.py

```diff
@@ -77,6 +79,10 @@ def test_logical_types():
 
 
 def test_not_logical_ints():
+    # This test doesn't work on Windows with python 3.6
+    if os.name == 'nt' and sys.version_info[:2] == (3, 6):
+        pytest.skip('Python Bug: https://bugs.python.org/issue29097')
```

Comment:
If you use `skipif`, you can pass the condition as part of the `skipif` function call. I think what you have works, this is more of a style thing.

Reply:
https://docs.pytest.org/en/latest/skipping.html#id1